content/en/docs/gke/deploy/deploy-cli.md

```diff
@@ -33,6 +33,9 @@ one if you haven't already.
   IAP)](https://cloud.google.com/iap/docs/).
 
 
+Refer to [Understanding the deployment process 
+](#understanding-the-deployment-process) for a description about the deployment steps and artifacts.
```

Comment:
Suggestion:

- Small refactoring to replace "for a description of the process" bit, which looks a little bit "off"

```diff
- Refer to 
- [Understanding the deployment process](#understanding-the-deployment-process)
- for a description about the deployment steps and artifacts.
+ To understand the kfctl configuration and deployment process in more detail, 
+ refer to
+ [Understanding the deployment process](#understanding-the-deployment-process).
```

or

```diff
Refer to 
[Understanding the deployment process](#understanding-the-deployment-process)
- for a description about the deployment steps and artifacts.
+ for more information on the kfctl configuration and deployment process.
```

(or something along these lines with/without mentioning "artifacts")

Reply:
Both look good to me.

Reply:
@mnmami Ok! Please do go ahead and make the changes, then we can `ok-to-test` it and move forward. Thanks!

Reply:
@mnmami Hi! Let us know if you have any updates like commits with new changes, etc 👍

Reply:
@8bitmp3 I don't have more changes here. The above LGTM.

Reply:
Thanks for getting back @mnmami. I can't make changes to your PR, I think. Can you please make a few revisions, stage and push and we'll take a look?

Reply:
@8bitmp3 sure, done.